queries/wof.jinja2

```diff
@@ -12,7 +12,7 @@ SELECT
       'area', area::bigint,
       'min_zoom', min_zoom,
       'max_zoom', max_zoom,
-      'is_landuse_aoi', CASE WHEN is_landuse_aoi = TRUE THEN TRUE END,
+      'is_landuse_aoi', is_landuse_aoi,
```

Comment:
Sorry, I meant that instead of including them as `__places_properties__`, we'd construct a "tags-like" JSON in the general `__properties__` which contains all the rows of the table and then allow the YAML to sort out which ones we pass through. E.g: something like:

```sql
SELECT
  ...
  to_jsonb(w.*) - 'geometry' - 'wof_id' AS __properties__
  ...
FROM wof_neighbourhood w
```